profiles/profiling_pipeline.py

```diff
@@ -0,0 +1,72 @@
+"""
+Perform the profiling pipeline (defined in profile.py) given all plates.
+"""
+
+import os
+import pathlib
+import subprocess
+import pandas as pd
+
+# Load constants
+project = "2015_10_05_DrugRepurposing_AravindSubramanian_GolubLab_Broad"
+batch = "2016_04_01_a549_48hr_batch1"
+profile_dir = pathlib.PurePath(
+    "/home/ubuntu/bucket/projects/", project, "workspace/backend"
+)
+barcode_platemap_dir = pathlib.PurePath("../metadata/platemaps")
+
+# Load barcode platemap information
+barcode_platemap_file = pathlib.PurePath(barcode_platemap_dir, "barcode_platemap.csv")
+barcode_platemap_df = pd.read_csv(barcode_platemap_file)
+
+# Load platemap information
+platemap_dir = pathlib.PurePath(barcode_platemap_dir, "platemap")
+
+# Load plate information
+plate_dir = pathlib.PurePath(profile_dir, batch)
+plates = [x for x in os.listdir(plate_dir) if x.startswith("SQ")]
+
+# Load and check MOA information
+moa_file = pathlib.PurePath(
+    "../metadata/moa/repurposing_info_external_moa_map_resolved.tsv"
+)
+moa_df = pd.read_csv(moa_file, sep="\t")
+assert isinstance(
+    moa_df, pd.DataFrame
+), "Error, MOA file does not exist. Is the path updated?"
+
+# Process every plate
+for plate in plates:
+    print(f"Now processing... Plate: {plate}")
+    output_dir = pathlib.PurePath("backend", plate)
+    cell_count_dir = pathlib.PurePath("analysis", plate)
+
+    platemap_id = barcode_platemap_df.query(
+        "Assay_Plate_Barcode == @plate"
```

Comment:
Same comment as before (regarding f-strings). Feel free to ignore this comment if you decide to stay with the current syntax.

Reply:
see comment above